charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus.yaml

```diff
@@ -137,6 +137,10 @@ spec:
         # we mount the Shoot cluster's CA and certs
         - mountPath: /etc/prometheus/seed
           name: prometheus-kubeconfig
+{{- if hasKey .Values.alerting.auth_type "certificate" }}
+        - mountPath: /etc/prometheus/operator
```

Comment:
Is this certificate dynamically reloaded by Prometheus or do we need an operation annotation such that Prometheus restarts in case the cert changes?

Reply:
Prometheus has the configmap reloader sidecar, so it should not be a problem. I will double check to make sure.

Reply:
I just checked and Prometheus will dynamically reload the certificate if it changes.